### PR TITLE
List of phrases

### PR DESCRIPTION
**Pull request by MichiBerger**

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
+import GlobalStyles from '../src/GlobalStyles.js';
+
 export const parameters = {
-  actions: { argTypesRegex: "^on[A-Z].*" },
+  actions: { argTypesRegex: '^on[A-Z].*' },
```
Comment (Quaquaro) [inline .storybook/preview.js:4]: Nice action!

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+a {
+  text-decoration: none;
+  color: black;
+}
+li {
+  list-style-type: none;
+}
+ul {
+  margin: 0;
```
Comment (Quaquaro) [inline src/GlobalStyles.js:29]: If you want the list-style to be none then put it in the ul tag. Don't forget the [role="list"]

File: src/components/PhraseCard.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    </PhraseCardWrapper>
+  );
+}
+
+const PhraseCardWrapper = styled.article`
+  display: grid;
+  grid-template-columns:  25% 25% 25% 25%;
+  grid-template-rows: auto 1fr auto;
+  grid-template-areas:
+    'date date . .'
```
Comment (Quaquaro) [inline src/components/PhraseCard.js:17]: Nice choice for layout!

File: src/components/PhraseCardList.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  {
+    date: '06. Jun. 2021',
+    phrase: 'DasIstDASLängsteWortUmZuÜberprüfenObDasWord',
+  },
+];
+
+export default function PhraseCardList() {
+  return (
+    <section>
+      <PhrasesList aria-label='phrases'>
```
Comment (Quaquaro) [inline src/components/PhraseCardList.js]: You maybe forget the role="list"

**Review by Quaquaro — APPROVED** [review]
Nice work!

File: src/GlobalStyles.js (hunk 2 above)
Comment (MichiBerger) [reply src/GlobalStyles.js:29]: thanks ;)

File: src/components/PhraseCardList.js (hunk 4 above)
Comment (MichiBerger) [reply src/components/PhraseCardList.js]: just changed it :)

**Pull request merged**